modules/gitops-uninstall-keycloak.adoc

```diff
@@ -0,0 +1,23 @@
+[id="uninstalling-keycloak_{context}"]
+= Uninstalling Keycloak
+
+You can delete the Keycloak resources and its relevant configuration by removing the SSO field from Argo CD Custom Resource Spec. After removing, the SSO field will look like as shown below.
+
+[source,yaml]
+----
+  apiVersion: argoproj.io/v1alpha1
+  kind: ArgoCD
+  metadata:
+    name: example-argocd
+    labels:
+      example: basic
+  spec:
+    server:
+      route:
+       enabled: true
+----
+
+[NOTE]
+====
+Keycloak application created by this feature is currently not persistent. Incase of restarts, nny additional configuration created in ArgoCD Keycloak realm will be deleted.
```

Comment:
Minor tweaks to the Note (small spelling and grammatical errors):

`A Keycloak application created by this feature is currently not persistent. In case of restarts, any additional configuration created in ArgoCD Keycloak realm is deleted.`